$NetBSD: patch-ae,v 1.2 2007/04/22 07:17:30 briandealwis Exp $

As per <mail:20070417173510.GB18879@ryvius.pekin.waw.pl>, ensure
the same hash is generated between 32 abd 64 bit machines.

--- crm_str_funcs.c.orig	2007-02-17 07:42:42.000000000 -0600
+++ crm_str_funcs.c
@@ -103,7 +103,7 @@ unsigned long strnhash (char *str, long 
 {
   long i;
   // unsigned long hval;
-  long hval;
+  int32_t hval;
   unsigned long tmp;
 
   // initialize hval
